manifest/sourcefile.py

```diff
@@ -297,6 +297,32 @@ def content_is_ref_node(self):
         graph (i.e. if it contains any <link rel=[mis]match>"""
         return bool(self.references)
 
+    @cached_property
+    def css_flag_nodes(self):
+        """List of ElementTree Elements corresponding to nodes representing a
+        to a flag <meta>"""
+        if not self.root:
+            return []
+        return self.root.findall(".//{http://www.w3.org/1999/xhtml}meta[@name='flags']")
+
+    @cached_property
+    def css_flags(self):
+        """Set of flags specified in the file"""
+        rv = set()
+        for item in self.css_flag_nodes:
+            if "content" in item.attrib:
+                for flag in item.attrib["content"].split():
+                    rv.add(flag)
+        return rv
+
+    @cached_property
+    def content_is_css_manual(self):
+        """Boolean indicating whether the file content represents a
+        CSS WG-style manual test"""
+        if not self.root:
+            return None
+        return bool(self.css_flags & {"animated", "font", "interact", "paged", "speech", "userstyle"})
```

Comment:
No? If there's the intersection between the flags in the document and that set, it's a manual test?